crates/swc_css_codegen/src/lib.rs

```diff
@@ -2763,27 +2763,27 @@ fn minify_hex_color(value: &str) -> String {
             if length == 6 || chars[6] == b'f' && chars[7] == b'f' {
                 let mut minified = String::with_capacity(3);
 
-                minified.push((chars[0] as char).to_ascii_lowercase());
-                minified.push((chars[2] as char).to_ascii_lowercase());
-                minified.push((chars[4] as char).to_ascii_lowercase());
+                minified.push(chars[0] as char);
+                minified.push(chars[2] as char);
+                minified.push(chars[4] as char);
 
                 return minified;
             }
             // 8 -> 4
             else if length == 8 && chars[6] == chars[7] {
                 let mut minified = String::with_capacity(4);
 
-                minified.push((chars[0] as char).to_ascii_lowercase());
-                minified.push((chars[2] as char).to_ascii_lowercase());
-                minified.push((chars[4] as char).to_ascii_lowercase());
-                minified.push((chars[6] as char).to_ascii_lowercase());
```

Comment:
Forgot to remove it too, we don't need it, because we already normalize hex colors